pyuvsim/simsetup.py

```diff
@@ -409,20 +614,23 @@ def initialize_catalog_from_params(obs_params, input_uv=None):
                     vo_params["dec_column"] = source_params["dec_column"]
 
                 vo_params["reference_frequency"] = None
-                catalog = pyradiosky.read_votable_catalog(
-                    catalog, **vo_params, return_table=True
+                sky = pyradiosky.read_votable_catalog(
+                    catalog, **vo_params
```

Comment:
The function you're calling here is deprecated. Can you use `SkyModel.read_votable_catalog`?